monitor.py

```diff
@@ -141,7 +141,7 @@ def click_b2(channel):
 
 	if apIndx >= 0:
 		main_fun(998)
-		print ''.join(pwdLst) #TODO change password for real...
+		print (''.join(pwdLst)) #TODO change password for real...
```

Comment:
This one was crashing the project so badly. I'm sure it's because i'm running it on python3